src/node_url.cc

```diff
@@ -786,12 +792,20 @@ bool ToASCII(const std::string& input, std::string* output) {
 }
 #else
 // Intentional non-ops if ICU is not present.
-bool ToUnicode(const std::string& input, std::string* output) {
+bool ToUnicode(Environment* env,
+               const std::string& input,
+               std::string* output) {
+  ProcessEmitWarning(env,
+                     "Cannot convert to unicode when intl is disabled");
   *output = input;
   return true;
 }
 
-bool ToASCII(const std::string& input, std::string* output) {
+bool ToASCII(Environment* env,
+             const std::string& input,
+             std::string* output) {
+  ProcessEmitWarning(env,
+                     "Cannot convert to ASCII when intl is disabled");
```

Comment:
If we do this, let’s be more specific about what actually happens here – “ToUnicode” and “ToASCII” are very generic names, and mirroring them 1:1 isn’t really useful to end users (it doesn’t even tell them that this is an operation on domain names that’s failing).